resources/lib/tubecast/dial.py

```diff
@@ -18,8 +18,8 @@
         <device>
             <deviceType>urn:schemas-upnp-org:device:dail:1</deviceType>
             <friendlyName>{{ friendlyName }}</friendlyName>
-            <manufacturer>Google Inc.</manufacturer>
-            <modelName>Eureka Dongle</modelName>
+            <manufacturer>enen92</manufacturer>
```

Comment:
Thanks for contributing. Don't think I could figure it out myself.
I'd prefer not to use my nick on the manifest. Can you use Kodi/tubecast instead? I need to test this on my devices too.

Reply:
I have changed both values